user_tools/src/spark_rapids_tools/utils/util.py

```diff
@@ -349,3 +349,13 @@ def bytes_to_human_readable(cls, num_bytes: int) -> str:
             num_bytes /= 1024.0
             i += 1
         return f'{num_bytes:.2f} {size_units[i]}'
+
+    @classmethod
+    def convert_df_to_dict(cls, df: pd.DataFrame) -> dict:
```

Comment:
- I don't think this can really be a util function because it is very tailored to specific use-case. Perhaps if the method is more generic to return a dictionary of `{key -> {col1: val, col2: val,...}}`, then this would be a utility function.
- If we want to keep it, we can consider moving this method to a separate util class/file. for example `df_ustils.py` to be the seed for any other helpers that we need for the dataframes.

Reply:
This method is no longer required as the logic to parse the runtime has been moved to Scala code.